docs/user/connected-accounts.rst

```diff
@@ -103,6 +126,7 @@ Bitbucket
 
 For similar reasons to those above for GitHub, we request permissions for:
 
+* Administing your repositories (to add a read-only deployment key)
```

Comment:
```suggestion
* Administering your repositories (to add a read-only deployment key)
```
No idea what the original word here is :)

Reply:
Original word was "Administer", -- "Administing" sounds fun though :)